packages/circuit-ui/components/SelectorGroup/SelectorGroup.tsx

```diff
@@ -52,12 +56,80 @@ export interface SelectorGroupProps {
    * Whether the user can select multiple options.
    */
   multiple?: boolean;
+  /**
+   * Whether to layout Selectors in a row or a column. Default: 'horizontal'.
+   */
+  orientation?: 'vertical' | 'horizontal';
+  /**
+   * Size of the Selectors within the group. Default: 'mega'.
+   */
+  size?: SelectorSize;
+  /**
+   * Spacing between selectors in a group. Default: 'kilo'.
+   */
+  gapSize?: GapSize;
```

Comment:
Can you share a use case where the gap size would need to be different than the default?

Reply:
This was is coming from the current use-cases we have in the Dashboard (see #931). But now I see it feel like over-customization, nothing will really change if we switch to one default gap size in the Dashboard. Will remove this prop.